web/src/components/organisms/LandingOfflineSection/index.jsx

```diff
@@ -37,9 +43,14 @@ const BackgroundContainer = styled.div`
 `;
 
 const OfflineSection = () => {
+  const [opacity, setOpacity] = useState(0);
+  window.addEventListener('scroll', () => {
```

Comment:
@ThibautChamoux This should happen in a useEffect. Otherwise, it will be added on every render.

Reply:
I think we should remove this; it's quite laggy on some devices

Reply:
Actually, that might be the reason...

Reply:
i removed it